### ISSUE TITLE
Create 1.x branch from master

### ISSUE THREAD

**Issue by leifmadsen** [torian/ansible-role-filebeat#4]
Per another PR, there was a small discussion around branching master into a 1.x branch so that changes for Ansible 2.x could be implemented. I'm just opening an issue so this isn't lost.

Once that is done, I'll open a couple PRs for some changes I wanted to make that clean things up, but which are not backwards compatible to Ansible < 2.1.

Thanks!

**Comment by torian** [comment]
@leifmadsen I'll create a branch named `stable-1.x`, and a corresponding tag for it. People in need of backward compatibility with _Ansible 1.x_ will be able to use this branch.

**Comment by leifmadsen** [comment]
w00t! Thanks! This is perfect.
